src/ansys/sherlock/core/launcher.py

```diff
@@ -132,13 +143,27 @@ def connect_grpc_channel(port=SHERLOCK_DEFAULT_PORT):
     return SHERLOCK
 
 
-def _get_base_ansys():
+def _get_base_ansys(year: int = None, release_number: int = None) -> str:
     supported_installed_versions = {
         env_key: path
         for env_key, path in os.environ.items()
         if env_key.startswith("AWP_ROOT") and os.path.isdir(path)
     }
 
+    if year is not None and release_number is not None:
+        try:
+            year = _extract_sherlock_version_year(year)
+            version_key = f"AWP_ROOT{year}{release_number}"
+            if version_key in supported_installed_versions:
+                return supported_installed_versions[version_key]
+            else:
+                raise ValueError(
+                    f"Specified version AWP_ROOT{year}{release_number} is not installed."

```

Comment:
Even though this is the mechanism used for finding the installed version, I think it is a bit cryptic to have the error formatted this way for an end user.
My recommendation is to change the wording to say:
"Sherlock {year} {release_number} is not installed."